jenkins/e2e-image/Makefile

```diff
@@ -15,18 +15,46 @@
 IMG = gcr.io/k8s-testimages/kubekins-e2e
 TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 
+K8S ?= master
+
+GO ?= 1.8.3
```

Comment:
[this should probably be 1.8.4 now](https://groups.google.com/forum/m/#!topic/golang-nuts/sHfMg4gZNps)?
/cc @ixdy

Reply:
Discussed offline, we will probably have to come back to this later. upgrading this everywhere will be tricky unfortunately, but the bugs fixed shouldn't be a problem in CI at least because nothing *should* but `go get`ing anything or using SMTP...

Reply:
I'm not sure we even need go in kubekins-e2e anymore. it was originally there so we could `go run hack/e2e.go`.

Reply:
hummm... node test still need go though, maybe some other stuff need go for something? (i.e. federation, kubemark), better keep it for now.